### ISSUE TITLE
v6 — error on displaying form in `6.0.0-alpha.15`

### ISSUE THREAD

**Issue by mayacode** [erikras/redux-form#1129]
MyForm.Component.js
```js
import React from 'react';
import { Field, reduxForm } from 'redux-form';
import MenuItem from 'material-ui/MenuItem';
import RaisedButton from 'material-ui/RaisedButton';
import baseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
import getMuiTheme from 'material-ui/styles/getMuiTheme';
import { SelectField } from 'redux-form-material-ui';

class MyForm extends React.Component {
  getChildContext() {
    return { muiTheme: getMuiTheme(baseTheme) };
  }
  
  render() {
    return (
      <form>
        <fieldset name="status">
          <div>
            <Field
              name="status"
              component={SelectField}
              hintText="Status"
              floatingLabelText="Status">
              <MenuItem value="0" primaryText="inactive"/>
              <MenuItem value="1" primaryText="active"/>
            </Field>
          </div>
        </fieldset>
      </form>
    );
  }
}

MyForm.childContextTypes = {
  muiTheme: React.PropTypes.object.isRequired
};

export default reduxForm({
  form: 'MyForm'
})(MyForm);
```

MyForm.Container.js
```js
import React from 'react';
import { connect } from 'react-redux';
import MyForm from '../components/MyForm';
import { postData, getData } from '../actions';
import PageContainer
  from '../../../modules/PageContainer/components/PageContainer';

class MyFormContainer extends React.Component {
  constructor(props) {
    super(props);
  }

  componentWillMount() {
    const { id } = this.props.routeParams;

    if (id !== undefined) {
      this.props.getData(id);
    }
  }

  render() {
    return (
      <PageContainer className='page-form' useContainer={true}>
        <MyForm {...this.props} onSubmit={this.props.postData }/>
      </PageContainer>
    );
  }

}

const mapStateToProps = (state) => {
  return {
    myForm: state.form.MyForm
  };
};

const mapDispatchToProps = (dispatch) => {
  return {
    postData: (data) => dispatch(postData(data)),
    getData: (data) => dispatch(getData(id))
  };
};

export default connect(mapStateToProps, mapDispatchToProps)(MyFormContainer);
```

`MyFormContainer` is a container bounded to a route in react router.

displaying this page I'm getting the error: `reduxForm.js:543 Uncaught TypeError: Cannot read property 'wrapped' of undefined`. In `6.0.0-alpha.4` there was no such error.

I would also appreciate the hint, what is wrong in this code. If I use components from `redux-form-material-ui` I can see them in `form` store in `MyForm` but if I use just components from `material-ui` (like DatePicker) cannot find a way to bound them to the form...

```js
"react": "15.1.0",
"react-tap-event-plugin": "1.0.0",
"redux": "3.0.5",
"redux-form": "^6.0.0-alpha.15",
"redux-form-material-ui": "^2.0.0",
```

**Comment by ooflorent** [comment]
Could you provide the complete error stack trace?

**Comment by halayli** [comment]
I am seeing the same issue with .15 but not in .14.

**Comment by uno4ki** [comment]
I confirm, this is react-hot-api and react-hot-loader related issue.

I've temporarily disabled HMR in my app and ll try to investigate this.

PS: Looks like problem with whole v6 branch (A10 is not working too)

**Comment by mayacode** [comment]
Sorry for delay, I was sick and didn't check my emails. So yes I confirm, that this problem appears when I use hot-loader.

**Comment by uno4ki** [comment]
I've updated to react-hot-loader@3.0.0-beta.1 and everything is working fine.

**Comment by qnm** [comment]
Upgrading `react-hot-loader@3.0.0-beta.1` worked for me too.

@erikras could we have a note about requiring the latest `react-hot-loader` added to the release notes - or perhaps a 'Known Issues' section on the migration guide?

Sadly `npm` `package.json` doesn't have support for 'optional dependencies' which would be the ideal place to declare this coupling.

**Comment by SpainTrain** [comment]
> or perhaps a 'Known Issues' section on the migration guide

:+1: #1566

**Comment by erikras** [comment]
Merged #1566. Closing this.
